diff --git a/vendor/bitwuzla/src/api/c/bitwuzla.c b/vendor/bitwuzla/src/api/c/bitwuzla.c
index 70a3a55d..55cc1ad1 100644
--- a/vendor/bitwuzla/src/api/c/bitwuzla.c
+++ b/vendor/bitwuzla/src/api/c/bitwuzla.c
@@ -3390,6 +3390,19 @@ bitwuzla_sort_is_rm(const BitwuzlaSort *sort)
   return bzla_sort_is_rm(bzla, bzla_sort);
 }

+void
+bitwuzla_sort_dump(const BitwuzlaSort *sort, FILE *file)
+{
+  BZLA_CHECK_ARG_NOT_NULL(sort);
+  BZLA_CHECK_ARG_NOT_NULL(file);
+
+  Bzla *bzla           = BZLA_IMPORT_BITWUZLA(sort->d_bzla);
+  BzlaSortId bzla_sort = BZLA_IMPORT_BITWUZLA_SORT(sort);
+  assert(bzla_sort_is_valid(bzla, bzla_sort));
+
+  bzla_dumpsmt_dump_sort(bzla_sort_get_by_id(bzla, bzla_sort), file);
+}
+
 /* -------------------------------------------------------------------------- */
 /* BitwuzlaTerm                                                               */
 /* -------------------------------------------------------------------------- */
diff --git a/vendor/bitwuzla/src/api/c/bitwuzla.h b/vendor/bitwuzla/src/api/c/bitwuzla.h
index b1f84a53..117f3488 100644
--- a/vendor/bitwuzla/src/api/c/bitwuzla.h
+++ b/vendor/bitwuzla/src/api/c/bitwuzla.h
@@ -3123,6 +3123,14 @@ bool bitwuzla_sort_is_fun(const BitwuzlaSort *sort);
  */
 bool bitwuzla_sort_is_rm(const BitwuzlaSort *sort);

+/**
+ * Print sort in the SMT-LIB v2 format.
+ *
+ * @param sort The sort.
+ * @param file The file to print the `sort` to.
+ */
+void bitwuzla_sort_dump(const BitwuzlaSort *sort, FILE *file);
+
 /* -------------------------------------------------------------------------- */
 /* BitwuzlaTerm                                                               */
 /* -------------------------------------------------------------------------- */
